manifest.json

```diff
@@ -6,11 +6,11 @@
         "en": "Hosting platform made for podcasters",
         "fr": "Plateforme d'hébergement conçue pour les podcasteurs"
     },
-    "version": "1.0.0-80~ynh1",
-    "url": "https://podlibre.org/",
+    "version": "1.0.0-5~ynh1",
```

Comment:
Eeeeh it sounds like we're moving backwards in terms of version number ?

Reply:
Damn yes... We went from alpha (1.0.0-alpha.80) to beta (1.0.0-beta.5) ...